roms_tools/setup/nesting.py

```diff
@@ -94,14 +105,24 @@ def _map_child_boundaries_onto_parent_grid_indices(self):
             self.metadata["period"],
         )
 
+        if verbose:
+            logging.info(f"Total time: {time.time() - start_time:.3f} seconds")
+            logging.info(
+                "========================================================================================================"
```

Comment:
Consider using another constant or even an int to avoid having these long, dupe lines.

```python
HEADER_SZ = 80
HEADER_CHAR = "="

@classmethod
def _log_header(cls) -> str:
    return cls.HEADER_CHAR * cls.HEADER_SZ
```

or 

```python
@classmethod
def log_the_separator(cls) -> None:
    logging.info(cls.HEADER_CHAR * cls.HEADER_SZ)
```

Reply:
9ca7046